patches/0002-Add-crypto-backend-foundation.patch

```
@@ -854,6 +886,14 @@ index 00000000000000..08600a2c833ac7
 +func VerifyEd25519(pub *PublicKeyEd25519, message, sig []byte) error {
 +	panic("cryptobackend: not available")
 +}
++
++func SupportsDSA(l, n int) bool {
++	return false
```

Why `return false` rather than `panic("cryptobackend: not available")`? (This is the only `return` in the file.)

---

fixed! Thanks for pointing this!